### PR TITLE
Use better language for a variable

### PR DESCRIPTION
**Pull request by kaicataldo**
Small PR to rename this variable, since I noticed it. No user-facing changes.

### CODE REVIEW

File: get-browser-globals.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const blacklist = [
+const denylist = [
```
Comment (sindresorhus) [inline get-browser-globals.js:3]: I don't think `deny` is the correct word here.
```suggestion
const ignorePatterns = [
```

Comment (kaicataldo) [reply get-browser-globals.js:3]: Makes sense to me!

File: get-browser-globals.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const blacklist = [
 const globals = Object.getOwnPropertyNames(window)
 	.sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()))
 	.filter(global => {
-		for (const pattern of blacklist) {
+		for (const pattern of denylist) {
```
Comment (kaicataldo) [inline get-browser-globals.js:48]:
```suggestion
		for (const pattern of ignorePatterns) {
```

Comment (kaicataldo) [reply get-browser-globals.js]: Ha, was able to batch the following change using this comment.

**Comment by kaicataldo** [comment]
Thanks! 🎉

**Pull request merged**